docker/Dockerfile

```
@@ -0,0 +1,75 @@
+# syntax = docker/dockerfile:1.2
+
+ARG BASE_IMAGE="ubuntu:20.04"
+FROM ${BASE_IMAGE}
```

I think it would be easier and more flexible to parameterize this on the ROS distro and build off of the nominal ROS docker image. Then there would be no need to install ROS and do a lot of other things below

```suggestion
ARG DISTRO="foxy"
FROM ros:${BASE_IMAGE}
```

---

Nice. I didn't realize that was an option. Will look into it.